CHANGES.rst

```diff
@@ -104,6 +104,8 @@ Improvements
 - Show the Indico version in the footer again (:issue:`4558`)
 - Event managers can upload a custom Book of Abstract PDF (:issue:`3039`,
   :pr:`4577`)
+- Allow registrants to withdraw their application (:issue:`2715`, thanks
+  :user:`brabemi` & :user:`omegak`)
```

Comment:
I'd also reference the PR since sphinx-issues supports that as well nowadays:

```suggestion
- Allow registrants to withdraw their application (:issue:`2715`, :pr:`4585`,
  thanks :user:`brabemi` & :user:`omegak`)
```